README.md

```diff
@@ -282,11 +282,21 @@ CDTQResultSet *result = [datastore find:query];
 }];
 ```
 
+```swift
+let query = [ "name" : "John", // name equals John
+    "age" : ["$gt" : 25] // age greater than 25
+]
```

Comment:
To be consistent with the Objective C example above (L275-278), this Swift example should read:
```swift
let query = [
    "name" : "John",     // name equals John
    "age" : ["$gt" : 25] // age greater than 25
]
```